libs/community/langchain_google_community/bigquery.py

```diff
@@ -56,7 +56,8 @@ def load(self) -> List[Document]:
         except ImportError as ex:
             raise ImportError(
                 "Could not import google-cloud-bigquery python package. "
-                "Please install it with `pip install google-cloud-bigquery`."
+                "Please, install bigquery dependency group: "
+                "poetry install --with bigquery"
```

Comment:
`pip install langchain-google-community[bigquery]` should be better, or am I missing anything?

Reply:
you are right, changing them

Reply:
done!